vmdb/spec/models/ems_refresh/refreshers/kubernetes_refresher_spec.rb

```diff
@@ -46,7 +48,7 @@ def assert_specific_container
       # :ems_ref       => "a7566742-e73f-11e4-b613-001a4a5f4a02_heapster_kubernetes/heapster:v0.9",
       :name          => "heapster",
       :restart_count => 0,
-      :image         => "kubernetes/heapster:v0.9",
+      :image         => "kubernetes/heapster:v0.11.0",
```

Comment:
Curiosity: do we intend to support multiple versions of heapster or only the latest?